dracut/anaconda-nfsrepo-cleanup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# NFS mount can sometimes cause a hang, a delay, or a timeout problem after
+# switching root, since it needs to survive the network reconfiguration in
+# stage2 phase. So, it's better to umount the unneeded NFS mounts before
+# switching root.
+
+# When rd.live.ram boot parameter, the stage2 image is copied into memory
+# from the NFS repository. So, keeping NFS mount is not required after
+# the copy. The NFS repository can be mounted in the stage2 phase of package
+# installation if needed.
+
+type getargbool >/dev/null 2>&1 || . /lib/dracut-lib.sh
+
+if getargbool 0 rd.live.ram -d -y live_ram; then
+    while read -r src mnt fs rest || [ -n "$src" ]; do
+        if [ "$mnt" = "/run/install/repo" ]; then
+            if [ "$fs" = "nfs" ] || [ "$fs" = "nfs4" ]; then
+                umount /run/install/repo
+                break
+            fi
+        fi
+        # inst.repo=nfs://dvd.iso case
+        if [ "$mnt" = "/run/install/isodir" ]; then
+            if [ "$fs" = "nfs" ] || [ "$fs" = "nfs4" ]; then
+                umount /run/install/repo
```

Comment:
I wonder if this couldn't hurt our future intentions to create source which would use Dracut repo mount? What do you think @poncovka ?
So or so it is just a future plan which may end up differently. I'm just thinking aloud, don't want to block this PR because of that.

Reply:
We should always set up the source defined by `inst.repo` in the stage2 phase, so that should be fine. Which installation source do we try to use with `inst.stage2=nfs://`? I don't think we fall back to `inst.stage2` in that case, so that should be fine as well.

Reply:
For inst.stage2=nfs:// closest mirror is used (or RedHat CDN for rhel) so I think we are fine.

Reply:
Shouldn't we look on the NFS path in case ``inst.stage2=nfs://`` without `inst.repo` specified? However, when we won't find any mount in Dracut everything should be fine because we will then work with the normal NFS mount? Or we can just don't care in this case and fallback to `Closest mirror` but than we again creating some special use-case. So or so, this have to be discussed later when we focus on the feature in more detail.